docs/vsphere-golden-image.md

```diff
@@ -129,6 +129,17 @@ Install the `docker` container runtime following the [Microsoft's documentation]
 Alternatively, you can run the provided PowerShell script [install-docker.ps1](vsphere_ci/scripts/install-docker.ps1) 
 to programmatically install `docker` in the Windows VM.
 
+### Set up for disconnected network environment
```

Comment:
Could someone using another cloud provider also be interested in this step? Just wondering if this should be in a more generic place (like the top-level README)

Reply:
Good point, given the creation of a golden image is only a pre-req for vSphere; here is definitely a good fit for this.
As per OpenShift [docs](https://docs.openshift.com/container-platform/4.8/installing/installing-preparing.html#supported-installation-methods-for-different-platforms), _restricted network_ is also supported in AWS with IPI, not the case for Azure.